src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/controller/dto/CreateFineAmount.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto
+
+import java.math.BigDecimal
+
+data class CreateFineAmount(
+  val sentence: CreateSentence,
```

Comment:
instead of this add a `val fineAmount: CreateFineAmount?` field in the https://github.com/ministryofjustice/hmpps-remand-and-sentencing-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/controller/dto/CreateSentence.kt object